src/inject/settings/hotkeys.js

```diff
@@ -9,13 +9,36 @@ const MODIFIER_KEYS = {
   18: 'Alt',
 };
 
-const ACTION_KEYS = _.transform(_.range(26), (final, current) => {
+const ACTION_KEYS = _.transform(_.range(26), (final, current) => { // letters
   final[current + 65] = String.fromCharCode(current + 65); // eslint-disable-line
+  if (current < 12) {
```

Comment:
Tagging the F keys and number keys into this function is really strange and I don't like it 😆 

Can you separate them into their own transforms